docassemble/ALToolbox/data/questions/al_income.yml

```diff
@@ -89,4 +89,602 @@ data: !!omap
   - charity: "Church or charitable donations"
   - loan payments: "Loan, credit, or lay-away payments"
   - support: "Support to someone not in household"
-  - other: "Other"
\ No newline at end of file
+  - other: "Other"
+---
+########## Questions
+---
+generic object: ALItemizedValueDict
+continue button field: x.revisit
+question: |
+  Edit values
+subquestion: |
+  ${ x.table }
+
+  ${ x.add_action() }
+---
+generic object: ALItemizedValueDict
+table: x.table
+rows: x
+columns:
+  - Type: |
+      row_index
+  - Amount: |
+      '$0' if hasattr(row_item, 'exists') and not row_item.exists else currency(row_item.value)
+edit:
+  - exists
+---
+generic object: ALItemizedJob
+code: |
+  x.source
+  x.employer.name.first
+  x.times_per_year
+  x.to_add.complete_attribute = 'complete'
+  x.to_subtract.complete_attribute = 'complete'
+  if x.is_part_time:
+    x.to_add["part time"].is_hourly = x.is_hourly
+  else:
+    x.to_add["full time"].is_hourly = x.is_hourly
+  x.to_add.gather()
+  x.to_subtract.gather()
+  x.complete = True
+---
+generic object: ALItemizedJob
+code: |
+  x.to_add[i].value
+  x.to_add[i].complete = True
+---
+generic object: ALItemizedJob
+code: |
+  x.to_subtract[i].value
+  x.to_subtract[i].complete = True
+---
+id: itemized job name
+generic object: ALItemizedJob
+question: |
+  What is your job title?
+fields:
+  - Job name: x.source
+---
+sets:
+  - x.employer.name.first
+id: employer
+generic object: ALItemizedJob
+question: |
+  Tell us who employs you as a ${ x.source }
+fields:
+  - Name: x.employer.name.first
+  - Street: x.employer.address.address
+    required: False
+  - Unit: x.employer.address.unit
+    required: False
+  - City: x.employer.address.city
+    required: False
+  - State: x.employer.address.state
+    required: False
+  - Zip: x.employer.address.zip
+    required: False
+  - Phone number: x.employer.phone_number
+    required: False
+---
+id: itemized job line items
+generic object: ALItemizedJob
+question: |
+    Enter info from your paystub for your job as a ${ x.source }
+fields:
+    - ${ word("What is your hourly pay for this job?") if x.is_hourly else word("What are your wages for this job?") }: x.to_add['full time'].value
+      datatype: currency
+      show if:
+        code: x.is_part_time is False
+    - ${ word("What is your hourly pay for this job?") if x.is_hourly else word("What are your wages for this job?") }: x.to_add['part time'].value
+      datatype: currency
+      show if:
+        code: x.is_part_time is True
```

Comment:
I don't think we should use the `word()` function here--it's a totally different translation system from the XLSX one. Maybe use the `label: field:` variation so there is room to use a Mako `if` statement and this will go to the regular translation file?

Reply:
never realized that those were separate, thanks. Do you mean it should change to something like this?:
```yaml
  - label: |
      % if x.is_hourly:
      What is your hourly pay for this job?
      % else:
      What are your wages for this job?
      % endif
  ```